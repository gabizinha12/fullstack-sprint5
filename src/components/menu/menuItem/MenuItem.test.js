import { render, screen } from "@testing-library/react";
import MenuItem from "./MenuItem";

describe("Teste de snapshot MenuItem", () => {
  it("should render MenuItem", () => {
    const { container } = render(<MenuItem link="teste" label="abc" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
