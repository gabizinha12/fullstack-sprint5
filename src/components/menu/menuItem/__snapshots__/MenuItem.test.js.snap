// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste de snapshot MenuItem should render MenuItem 1`] = `
<li
  class="sc-bdnxRM"
>
  <a
    class="sc-gtsrHT ijhrOz"
    href="teste"
  >
    <span
      class="sc-dlnjwi"
    >
      abc
    </span>
  </a>
</li>
`;
